camera: 
  translation:
    x: 0
    y: 0
    z: 0

box: 
  minSize: 2000
  maxAreaIndex: 0.2
  maxEigenIndex: 1.069
  maxDiagIndex: 75

flags: 
  areaCheck: true
  diagCheck: true
  eigenCheck: true
  debug: true
  verbose: false

expectedSizes: 
  red: 0
  yellow: 0
  blue: 0 
  orange: 0